rubrik_cdm/cluster.py

```diff
@@ -606,3 +608,58 @@ def read_only_authorization(self, username, timeout=15):
 
         self.log("read_only_authorization: Granting read-only privilages to user '{}'.".format(username))
         return self.post("internal", "/authorization/role/read_only_admin", config, timeout)
+
+    def cluster_node_id(self):
+        """ Returns a list of node ids from all the nodes in the cluster.
+
+                Arguments:
+                    None
+
+                Keyword Arguments:
+                    None
+
+                Returns:
+                    dict -- The full API response from `POST /internal/authorization/role/read_only_admin`.
+                """
+        self.log("cluster_node_id - Getting all the node ids from the from the cluster")
+        node_id = self.get('internal', '/node')
+        return node_id
+
+    def cluster_support_tunnel(self,action):
+        """ Function that can check the status, enable or disable the support tunnel.
+
+                        Arguments:
+                            Action - with state of (Status,Enable,Disable)
+
+                        Keyword Arguments:
```

Comment:
Keyword Arguments should only be the arguments that have a default value. For example, `timeout=10`. So you can just put a description of each action here `Action - with state of (Status,Enable,Disable)`